entities/license/directoryOffering.json

```diff
@@ -74,6 +74,8 @@
     },
     "required": [
         "directory_offering_uid",
+        "directory_uid",
```

Comment:
Why is directory_uid required? I am not sure this is the right approach. It will also mess with migration and we may want to utilize null here.